checkov/common/runners/base_runner.py

```diff
@@ -1,7 +1,12 @@
+import os
+
 from abc import ABC, abstractmethod
 
 from checkov.runner_filter import RunnerFilter
 
+IGNORED_DIRECTORIES_ENV = os.getenv('IGNORED_DIRECTORIES',"node_modules,.terraform,.serverless")
```

Comment:
I would add this env variable to docs, explaining it's a comma separated string of ignored directories.

Reply:
`CK_IGNORED_DIRECTORIES` something with a prefix